mindboggle/mindboggle123

```diff
@@ -87,6 +87,9 @@ adv_args.add_argument("--fs_openmp", dest="openmp",
 adv_args.add_argument("--fs_T2image", dest="T2image",
                       type=str,
                       help="Optional T2 image to use with FreeSurfer")
+adv_args.add_argument("--fs_3T", dest="T3",
```

Comment:
@PeerHerholz -- Will you be updating your pull request to incorporate @satra's suggestion for a more generic string argument?